.papr.inventory

```diff
@@ -7,14 +7,25 @@ etcd
 ansible_ssh_user=root
 ansible_python_interpreter=/usr/bin/python3
 openshift_deployment_type=origin
-openshift_image_tag="{{ lookup('env', 'OPENSHIFT_IMAGE_TAG') }}"
+openshift_image_tag="{{ lookup('env', 'OPENSHIFT_IMAGE_TAG') | default(None) }}"
```

Comment:
Unfortunately, type None is not the same as 'undefined' in ansible, which is what some of our logic uses.

Perhaps default to openshift_release's value.

Reply:
Using openshift_release here might not be what we want long term, though.

Reply:
actually, i may be able to just strip this out now